### ISSUE TITLE
Strange rendering of leisure=track

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1900]
**[Submitted to the original trac issue database at 6.54am, Tuesday, 2nd June 2009]**

I think it would be a good idea to change the rendering of leisure=track, because tracks are not always in the form of a stadium track. For an example, look at http://www.openstreetmap.org/?lat=63.842&lon=20.3651&zoom=14&layers=0B00FTF . The track is a runners (skiing in winter) track in the forest - quite common in Sweden - consisting of several entangled loops, making the area fill look very strange. To make matters worse, the area contains several paths (most of them are not mapped yet) that are not part of the runners track, making it hard to see which paths belong to the track, and which don't.

I don't know if these designated runners/skiers tracks are common elsewhere, but this is how they typically look: http://sv.wikipedia.org/wiki/Elljussp%C3%A5r . They are not just paths in the woods; they are designated exercise tracks with markings, measured length and so on.

**Comment by tomhughes** [comment]
**Author: petschge**
**[Added to the original trac issue at 12.27pm, Thursday, 11th June 2009]**

(In [15845]) Remove area fill from leisure=track. Fixes #1900
Also fix some whitespace

**Comment by tomhughes** [comment]
**Author: schuetzm@gmx.net**
**[Added to the original trac issue at 9.53pm, Friday, 17th July 2009]**

It would be nice if area=yes were honored.

**Comment by tomhughes** [comment]
**Author: osm@petschge.de**
**[Added to the original trac issue at 5.07pm, Saturday, 18th July 2009]**

If the way is closed it should have area fill anyway. and if the way is not closed, honoring area=yes might give funny results. So why would you need that?

**Comment by tomhughes** [comment]
**Author: schuetzm@gmx.net**
**[Added to the original trac issue at 8.56am, Sunday, 19th July 2009]**

Many tracks are closed ways, as they often go around a stadium, e.g. here:
http://www.openstreetmap.org/?lat=49.9004&lon=10.92902&zoom=17&layers=0B00FTF
So it makes sense not to automatically make them filled when they are closed.

But here is an example where I entered the outline of a rectangular track:
http://www.openstreetmap.org/?lat=49.95687&lon=10.87213&zoom=17&layers=0B00FTF
In this particular case, it would probably ok to use a simple straight line instead of a rectangle, but still...

Basically it's the same as with street areas vs. street loops.

**Comment by tomhughes** [comment]
**Author: petschge**
**[Added to the original trac issue at 12.02pm, Sunday, 26th July 2009]**

(In [16689]) Honor area=yes on leisure=track. Fixes #1900.

Add leisure=track with and without area=yes to stadion.osm for testing.
